protos/flyteidl/plugins/pytorch.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package flyteidl.plugins;
+
+option go_package = "github.com/lyft/flyteidl/gen/pb-go/flyteidl/plugins";
+
+message PyTorchOperatorTask {
```

Comment:
Can you add some code comments here...

Reply:
Sure, done.

Reply:
I think we should call the plugin type, DistributedPytorchTraining?

Reply:
Also the comment then should be, this plugin enables distributed training for pytorch using "link to the Kubeflow operator" - What do you think @igorvalko ?

Reply:
done